padertorch/summary/tbx_utils.py

```diff
@@ -210,14 +263,56 @@ def audio(signal, sampling_rate: int = 16000, batch_first=False,
     return signal, sampling_rate
 
 
+def matplotlib_figure_to_image(
+        figure: 'matplotlib.figure.Figure' = None,
+) -> np.ndarray:
+    """
+    Converts a matplotlib figure to a numpy array that can be handled by
+    tensorboardX.
+
+    Note:
+        Matplotlib is in general quite slow, so you shouldn't be computing
+        matplotlib plots in every iteration. Use the `pt.Model.create_snapshot`
+        flag to reduce the computational overhead.
+
+    Warnings:
+        This function doesn't close the plot! Make sure that you close unused
+        plots to reduce memory consumption!
+
+    Args:
+        figure: matplotlib figure object. If `None`, it defaults to `plt.gcf()`.
+
+    Returns:
+        Numpy array with shape (color (4), width, height)
+
+    Examples:
+        >>> from matplotlib import pyplot as plt
```

Comment:
Add lines, how to disable GUI backend?
e.g.
`>>> # import matplotlib as mlp`
`>>> # mlp....  # Disable GUI backend, must be executed before the first "import matplotlib.pyplot"`

Reply:
Not required anymore